### ISSUE TITLE
I'm finding the 2.0.0 font less readable than 1.3.2's

### ISSUE THREAD

**Issue by kreducer** [oakes/Nightcode#173]
This one's a bit subjective and probably biased by the fact that I've been enjoying coding in 1.3.2 so much, but I'm finding the choice of a different font in the NC2 editor is making reading code harder. The &/@ symbols in particular don't appear in the shape I'm expecting to see and the characters are overall a bit more packed and less legible than before at the same font size. It'd be handy if there was the possibility for users to select a font face as well as size, but IMO the old NC font is more legible.

* NC 2.0.0 (from windows installer exe on the site)
* Windows 10
* System JRE 1.8.0_92-b14 (in case that's helpful).

**Comment by oakes** [comment]
Currently I'm using [FiraCode](https://github.com/tonsky/FiraCode) which is fairly popular for editing code. I may look into allowing the font to change in the future. I changed the title to reflect that.

**Comment by egri-nagy** [comment]
Yes, it is very subjective, but I am very attached to coding font I'm using (https://madmalik.github.io/mononoki/), so it would be great if I could use that with NC. Maybe just a few instructions how to build NC with a different font would be sufficient.

**Comment by egri-nagy** [comment]
By modifying `public/codemirror.html` and `public/paren-soup.html`, and uploading the font to `public/fonts` I could change the font for the code.
This is good enough for my use case.
